modules/prefixer.js

```diff
@@ -184,7 +184,15 @@ var _getPrefixedValue = function (property, value, originalProperty) {
     }
 
     if (typeof value !== 'string') {
-      value = value.toString();
+      if (value !== null && value !== undefined) {
+        value = value.toString();
+      } else {
+        /* eslint-disable no-console */
+        if (console && console.warn) {
+          console.warn('CSS value is "' + value + '" for property "' + property + '"');
```

Comment:
I think I might be missing something, but we don't have a reference to a `component` within `prefixer`? Should `getPrefixedStyle` be taking a `component` as an argument for the sake of adding the name in the warning?